py3status/modules/nvidia_temp.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
 """
-Display NVIDIA GPU temperature.
+Display NVIDIA GPU temperature(s).
 
 Configuration parameters:
     cache_timeout: how often we refresh this module in seconds (default 10)
-    format_prefix: a prefix for the output. (default 'GPU: ')
-    format_units: the temperature units. Will appear at the end. (default '°C')
-    temp_separator: the separator char between temperatures (only if more than
-        one GPU) (default '|')
+    format: display format for nvidia_temp (default 'GPU: {format_temp}')
+    format_temp: display format for gpu temperature(s) (default '{temp}°C')
+    icon_separator: display separator (only if more than one) (default '|')
+    msg_unavailable: display when unavailable (default 'GPU: N/A')
```

Comment:
Github should just stop collapse all outdated comments. Display everything as-is. I removed the prefix, but will leave `N/A`. I think people won't use it (the module) at all if they found out that it can't read the temperature. Just return direct `N/A` instead?